content/docs/command-reference/remote/add.md

```diff
@@ -344,6 +344,18 @@ $ dvc remote add -d myremote https://example.com/path/to/dir
 
 <details>
 
+### Click for WebDAV
+
+```dvc
+$ dvc remote add -d myremote webdavs://example.com/path/to/dir
```

Comment:
Should `/path/to/dir` be `/absolute/path` like in SSH and HDFS?

Reply:
It should be similar to `HTTP`, but could also contain the location of the WebDAV endpoint (e.g. `/public.php/webdav`). This might depend on the server configuration and could even differ for proper users vs. users created by public links for sharing.

It is not absolute like in `SSH` or `HDFS`, the most common approach will probably be to just give the location of the WebDAV API e.g. `webdavs://example.com/public.php/webdav`. Only if there are remotes in sub folders, additional `/path` would be required which is than relative to the user's root directory.

I do not know whether it is useful/possible to explain various options here or better leave it as simple as possible, so the  user should know how their WebDAV paths are configured?

Reply:
> but could also contain the location of the WebDAV endpoint (e.g. /public.php/webdav)...
> the most common approach will probably be to just give the location of the WebDAV API

Sounds like we should use:

```suggestion
$ dvc remote add -d myremote webdavs://example.com/public.php/webdav
```

And a brief explanation mentioning "WebDAV endpoint" (link to their docs if possible)

Reply:
> I do not know whether it is useful/possible to explain various options here or better leave it as simple as possible

Definitely keep in simple in `remote add`, but the `url` param should be fully explained in `remote modify`, with both examples.

> if there are remotes in sub folders, additional /path would be required

So should the path-based URL example (in `remote modify`) be
```
$ dvc remote add -d myremote \
                 webdavs://example.com/public.php/webdav/path/to/dir
```
? (note that after 72 chars we break lines in code blocks).